### ISSUE TITLE
Exposure to humid conditions

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9965]
**Preferred term label:**
Exposure to humid conditions

**Synonyms**
Excess moisture and heat, tropical, muggy

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Tinea (various forms)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Not sure this is easy to define--most areas have some humidity sometime. Is there a more precise definition that would allow users to unambiguously know when to use the term?

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding
HP:0020070
Exposure to humid conditions
Def: A history of living in an environment characterized by excess moisture and heat, tropical conditions, or muggy weather.
